tests/src/test_cases/test_snapshot.c

```diff
@@ -30,6 +35,7 @@ void test_snapshot_should_not_leak_memory(void)
 
     lv_mem_monitor(&monitor);
     final_available_memory = monitor.free_size;
+    lv_obj_delete(label);
 
     TEST_ASSERT_EQUAL(initial_available_memory, final_available_memory);
```

Comment:
Please add a separate screen shot test too for transformations, just to be sure.

Reply:
Where the screen shot test is added?

Reply:
`test_snapshot_with_transform_should_not_leak_memory` function is the test for transformation.

Reply:
> But there is no ` TEST_ASSERT_EQUAL_SCREENSHOT` in it so we can't see if a transformed widget was rendered correctly.

got it.